packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -36,13 +36,15 @@ class DatadogSpanContext {
         ? this._trace.tags[TRACE_ID_128] + this._traceId.toString(16).padStart(16, '0')
         : this._traceId.toString(16).padStart(32, '0')
     }
+    // if (!this._traceId) return ''
```

Comment:
Dead code should be removed and not just commented out.

Reply:
will do